src/core/gosling.schema.ts

```diff
@@ -209,6 +209,51 @@ export interface CommonTrackDef extends CommonViewDef {
 export interface DataTrack extends CommonTrackDef {
     data: DataDeep;
 }
+/**
+ * A placeholder track. In contrast to other tracks, this track does not display any data. Instead it provides
+ * empty space for third party tools to display their data on top of.
+ */
+export interface DummyTrack
+    extends Pick<
+        CommonTrackDef,
+        | 'width'
+        | 'height'
+        | 'id'
+        | 'title'
+        | '_invalidTrack'
+        | 'overlayOnPreviousTrack'
+        | 'orientation'
+        | 'layout'
+        | 'static'
+        | 'assembly'
+    > {
+    /** Used to specify the dummy track */
+    type: 'dummy-track';
+    /** Text that gets shown on the DummyTrack */
+    title?: string;
+    /** Defines how the track is styled */
+    style?: DummyTrackStyle;
+    /** Only linear layout are supported at this time */
+    layout?: 'linear';
+    static?: true;
+    /** Unused property for DummyTrack */
+    zoomLimits?: [null, null];
+    /** No assemblies can be associated with a dummy track */
+    assembly?: 'unknown';
+}
```

Comment:
Rather than just extending CommonTrackDef, I tried making making the properties used explicit so users won't be confused. Some of them like layout, static, and zoomLimits I kept in there they're heavily used in spec-preprocess.ts and I would have to significantly refactor it to not have them. Let me know if you think there would be a better way to do this.